package.xml

```diff
@@ -6,6 +6,10 @@
   <description>The package provides GoogleMock.</description>
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>BSD</license>
+  
+  <buildtool_depend>cmake</buildtool_depend>
+  
+  <buildtool_export_depend>cmake</buildtool_export_depend>
```

Comment:
It likely doesn't. I was tripped up by some of the cross compilation wording documenting the keys. I'll fix these up.